### ISSUE TITLE
[Accessibility]: Switching editor (Ctrl+Tab) does not work from a11y terminal buffer

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#177755]
CC @meganrogge

1. Enable screen reader.

1. In the same editor group, create first editor for source code and the second editor for terminal.

1. Now you can switch between two editors by pressing Ctrl+Tab.

1. However, Ctrl+Tab does not work in the a11y terminal buffer. It behaves like a tab key press.

* Note: What if users want to jump between source code editor and a11y terminal buffer?

## System Info

Version: 1.77.0-insider (system setup)
Commit: fa13bf16fd237afef07f4df0648da026f43e3963
Date: 2023-03-20T05:25:30.711Z
Electron: 19.1.11
Chromium: 102.0.5005.196
Node.js: 16.14.2
V8: 10.2.154.26-electron.0
OS: Windows_NT x64 10.0.22621
Sandboxed: Yes

**Comment by meganrogge** [comment]
We have to handle `Tab` specially in the accessible buffer, so this will be tricky to implement. I'll see if we can do something about it though

**Comment by aeschli** [comment]
Verified that Alt+Tab works when in the a11y terminal buffer.
